client/containers/DashboardOverview/overviewRows/CollectionOverviewRow.tsx

```diff
@@ -25,6 +25,37 @@ type Props = {
 	onToggleOpen?: () => unknown;
 };
 
+const labelPairs = (iconLabelPairs: IconLabelPair[]) => {
```

Comment:
I think this would have a natural home as a utility that gets imported from `labels.tsx` instead of duplicated across several files. It should also have a more verb-y name, maybe `renderLabelPairs`.